bugsnag-android-performance-impl/src/main/kotlin/com/bugsnag/android/performance/internal/BugsnagPerformanceImpl.kt

```diff
@@ -180,3 +183,12 @@ public object BugsnagPerformanceImpl {
         return spanControlProvider[query as SpanQuery<Any>] as C
     }
 }
+
+private fun setDebugTimout(
+    externalConfiguration: PerformanceConfiguration,
+    configuration: ImmutableConfig,
+) {
+    if ((externalConfiguration.isDevelopment ?: configuration.application.isDebuggable) == true) {
+        InternalDebug.workerSleepMs == 5000L
+    }
+}
```

Comment:
This should be a utility function within the `BugsnagPerformanceImpl` class, rather than being outside of the class